### ISSUE TITLE
test phase: why convert the label to [321, 481] ?

### ISSUE THREAD

**Issue by AceCoooool** [cswin/WIN#2]
Question about: Consistency
I am confusing about the explanation in Consistency.m file: `keep the size consistency between the test and training data` (the training data's size is 41x41 ?)

**Comment by rcrandall** [comment]
See https://github.com/cswin/WIN/issues/1. It appears that the provided model is overfit to a specific noise pattern, and the denoising performs very poorly if the image size changes.
